### PR TITLE
AB#8783 Feature/9063 password pipe

### PR DESCRIPTION
**Pull request by xchanin**

### CODE REVIEW

File: projects/common/src/lib/controls/api-access/api-access.component.ts [hunk 1]
```diff
@@ -40,6 +40,8 @@ export class ApiAccessComponent implements OnChanges, OnInit {
   @Output('regenerated')
   public Regenerated: EventEmitter<string>;
 
+  protected toggleKeyVisibility: boolean;
```
Comment (mcgear) [inline projects/common/src/lib/controls/api-access/api-access.component.ts:43]: move up to the //Fields section

File: projects/common/src/lib/controls/api-access/api-access.component.ts [hunk 2]
```diff
@@ -68,7 +70,9 @@ export class ApiAccessComponent implements OnChanges, OnInit {
   }
 
   public ShowKey(keyData: any) {
```
Comment (mcgear) [inline projects/common/src/lib/controls/api-access/api-access.component.ts:72]: alphabetical order

**Review by mcgear — APPROVED** [review]

File: projects/common/src/lib/controls/api-access/api-access.component.ts (hunk 1 above)
Comment (xchanin) [reply projects/common/src/lib/controls/api-access/api-access.component.ts:43]: We don't need this, just forgot to remove it.

File: projects/common/src/lib/controls/api-access/api-access.component.ts (hunk 2 above)
Comment (xchanin) [reply projects/common/src/lib/controls/api-access/api-access.component.ts:72]: I didn't add this property, so I didn't notice that.

**Pull request merged**